src/ape/api/accounts.py

```diff
@@ -258,7 +263,7 @@ def declare(self, contract: "ContractContainer", *args, **kwargs) -> ReceiptAPI:
 
     def check_signature(
         self,
-        data: Union[SignableMessage, TransactionAPI],
+        data: Union[SignableMessage, TransactionAPI, str, EIP712Message],
```

Comment:
I wonder if we should just make this `Any` as well.
Starknet signs ints. They love ints more than any other data type in that ecosystem. I want ints.

Reply:
ints have been added, I do agree this list is getting large